packages/server/lib/controllers/v1/plans/usage/getUsage.ts

```diff
@@ -12,12 +13,30 @@ export const getUsage = asyncWrapper<GetUsage>(async (req, res) => {
         return;
     }
 
-    const { account, plan } = res.locals;
-    if (!plan || plan.name !== 'growth') {
+    const { account, user, plan } = res.locals;
+    if (!plan) {
         res.status(400).send({ error: { code: 'feature_disabled' } });
         return;
     }
 
+    // Backfill orb customer
+    if (!plan.orb_customer_id) {
+        const linkOrbCustomerRes = await linkOrbCustomer(account, user);
+        if (linkOrbCustomerRes.isErr()) {
+            res.status(500).send({ error: { code: 'server_error', message: 'Failed to link Orb customer' } });
```

Comment:
```suggestion
            res.status(500).send({ error: { code: 'server_error', message: 'Failed to link billing customer' } });
```
No need to mention `orb` imho